packages/contracts/test/TestTwitter.t.sol

```diff
@@ -9,8 +9,6 @@ import "../src/Verifier.sol";
 contract TwitterUtilsTest is Test {
     using StringUtils for *;
 
-    address constant VM_ADDR = 0x7109709ECfa91a80626fF3989D68f67F5b1DD12D; // Hardcoded address of the VM from foundry
```

Comment:
I wasn't sure what this line was adding?

Reply:
My guess is that it was some hotfix for an old foundry verrsion that required this for tests to pass. If removing this still lets tests pass, then that seems fine to me!